CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to Open Liberty Docker
+
+Anyone can contribute to the Open Liberty Docker project and we welcome your contributions!
+
+There are multiple ways to contribute: report bugs, fix bugs, contribute code, improve upon documentation, etc. You must follow these guidelines:
+
+- [Raising issues](##-Raising-Issues)
+- [Contributor License Agreement](##-Contributor-License-Agreement)
+- [Coding Standards](##-Coding-Standards)
+
+## Raising Issues
+
+Please raise any bug reports on the Open Liberty Docker project repository's GitHub issue tracker. Be sure to search the list to see if your issue has already been raised.
+
+A good bug report is one that makes it easy for everyone to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
+
+## Contributor License Agreement
+
+In order for us to merge any of your changes into Open Liberty Docker, you need to sign the Contributor License Agreement. When you open up a Pull Request for the first time, a bot will comment with a link to the CLA. You can review or sign the CLA now here https://cla-assistant.io/OpenLiberty/ci.docker. You must verify your country has laws recognizing the use of legally binding e-Signatures, otherwise you must follow the hardcopy instructions [here](cla/open-liberty-cla-individual.pdf).
```

Comment:
Would it be reasonable to re-word this to say "When you open up a Pull Request for the first time, a bot will comment with a link to the CLA. You may also review and sign the CLA here: https://cla-assistant.io/OpenLiberty/ci.docker." ? If they open it and sign it, will it preclude the bot from commenting on their first issue?

Reply:
Yes if the user goes to the link https://cla-assistant.io/OpenLiberty/ci.docker and signs it the bot won't comment, I changed the wording to include that.